Add annotated examples for single/multiple fonts

RoelN opened the pull request:
Notably, a dropdown to change font with.

RoelN commented:
Note that the order of fonts is nondeterministic, and can change with every build. @pascalw Any way to work around this? I'm thinking of maybe normalizing the 11ty object into something easier to use, like a flat array. WDYT?

RoelN commented:
Also, not happy with duplicating the font selectors stuff. Maybe we should generate an extra _data file from the support, which contains the human readable font names, the CSS font name, and the CSS class, so a developer can pick the notation they need.

pascalw commented:
As discussed offline:

* The non deterministic order seems to be caused by 11ty. 11ty adds global data files to the global data file in an non-deterministic order. Even if it didn't, relying on ordering of object keys is not very safe. So best is probably to generate a list of fonts somewhere, where we control the order of that list.
* Duplicating the selector stuff can be prevented by including the selectors for each font in `fonts.js`.

RoelN commented:
I addressed the 2nd point and moved the name/classname generation to the support.

Regarding handling the nondeterministic stuff, I'll open an issue for this and address this in the near future!

RoelN commented:
Issue to address the 1st point: https://github.com/kabisa/specimen-skeleton/issues/64

pascalw commented on src/js/main.js:
```diff
@@ -38,7 +38,7 @@ const throttle = (fn, wait) => {
 // Set up FontFaceObserver
 let observers = [];
 for (const fontName of fontNames) {
```
Oh also don't forget to update the version of the support library!

RoelN replied:
Will change names, and will update refernce after merge!
